src/lib/Components/ArtworkGrids/ArtworkGridItem.tsx

```diff
@@ -55,25 +62,49 @@ export const Artwork: React.FC<Props> = props => {
         destinationScreenOwnerSlug: artwork.slug,
       })
 
-      trackTap ? trackTap(props.artwork.slug, props.itemIndex) : tracking.trackEvent(genericTapEvent)
+      trackTap ? trackTap(artwork.slug, itemIndex) : tracking.trackEvent(genericTapEvent)
     }
   }
 
-  const artwork = props.artwork
-  const artworkImage = artwork.image
-  const saleInfo = saleMessageOrBidInfo(artwork)
+  const saleInfo = saleMessageOrBidInfo({ artwork })
+
+  const renderSaleInfo = () => {
+    return (
+      <Sans color="black60" size="3t" numberOfLines={1}>
+        {saleInfo}
+      </Sans>
+    )
+  }
+
+  const urgencyTag = getUrgencyTag(artwork?.sale?.endAt)
 
   return (
     <Touchable onPress={() => handleTap()}>
       <View ref={itemRef}>
-        {!!artworkImage && (
-          <OpaqueImageView aspectRatio={artwork.image?.aspectRatio ?? 1} imageURL={artwork.image?.url} />
+        {!!artwork.image && (
+          <View>
+            <OpaqueImageView
+              aspectRatio={artwork.image?.aspectRatio ?? 1}
+              imageURL={artwork.image?.url}
+              style={styles.artworkImage}
+            >
+              {Boolean(urgencyTag && artwork?.sale?.isAuction && !artwork?.sale?.isClosed) && (
+                <Flex backgroundColor="white" px="3px" py="5px" borderRadius={2} alignSelf="flex-start">
```

Comment:
suggestion: i think you got these padding values the wrong way around

```suggestion
                <Flex backgroundColor="white" px="5px" py="3px" borderRadius={2} alignSelf="flex-start">
```